src/api/unicorns.service.ts

```diff
@@ -0,0 +1,32 @@
+import { IDatabase } from "../db";
+import { Unicorn, ResolvedUnicorn } from "./interfaces";
+import { MealsService } from "./meals.service";
+
+export class UnicornsService {
```

Comment:
i don't like that you use the string "unicorns" so much, either put it in a const (or inject in the constructor), or maybe create a concept of `collection` and not only `database`, and the service won't know the collection's name